_posts/06-02-01-Basic-Concept.md

```diff
@@ -48,6 +48,6 @@ class Database
 class MysqlAdapter {}
 {% endhighlight %}
 
-Sada klasi `Database` mi dajemo njenu zavisnost radije nego da ona to sama kreira. Mogli smo čak da napravimo
+Sada klasi `Database` dostavljamo njenu zavisnost umesto da je ona sama kreira. Mogli smo da napravimo i
 metod koji bi prihvatao argument zavisnosti i na taj način je postavljao, ili da je polje `$adapter` bilo
```

Comment:
'Bem li ga, možda sam slučajno izbacio, zato što kad sam prepravljao nisam išao reč po reč, već sam više prepisivao kompletnu rečenicu svojim rečima.